bycycle/features/features.py

```diff
@@ -83,13 +83,11 @@ def compute_features(sig, fs, f_range, center_extrema='peak', burst_method='cycl
         - ``monotonicity`` : fraction of instantaneous voltage changes between consecutive
           samples that are positive during the rise phase and negative during the decay phase
 
-        When dual threshold burst detection is used (i.e. burst_method == 'amp'):
+        When dual threshold burst detection is used (i.e. burst_method = 'amp'):
 
         - ``burst_fraction`` : fraction of a cycle that is bursting
 
-    df_samples : pandas.DataFrame, optional, default: True
-        An optionally returned dataframe containing cyclepoints for each cycle.
-        Columns (listed for peak-centered cycles):
+        When cyclepoints are returned (i.e. deafault, return_samples = True)
```

Comment:
typo on `deafault`

Reply:
also - in comments do / should we write `return_samples = True` or `return_samples=True`. 
It looks a little weird to me to write this differently in comment than we would in code.